go.mod

```diff
@@ -1,17 +1,17 @@
 module github.com/moby/moby/v2
 
-go 1.24.0
+go 1.24.3
 
 require (
-	cloud.google.com/go/compute/metadata v0.6.0
+	cloud.google.com/go/compute/metadata v0.7.0
```

Comment:
We can probably update some of these already ahead of time.

Reply:
Okay I'll start breaking them out but label them for 29.1.